source/tutorial/dotnet.txt

```diff
@@ -74,23 +74,61 @@ Prerequisites
 
 Start with the Template
 -----------------------
-This tutorial is based on the .NET Flexible Sync Template App named 
-"maui.todo.flex". We start with the default app and build new features 
-on it. 
 
-If you already have the latest Realm CLI installed, you can run the 
-:ref:`realm-cli apps create <realm-cli-apps-create>` command to set up the 
-backend and create the Xamarin base app. The following command creates a new 
-app based on the xamarin.todo.flex template. The app is created in a 
-development environment, is named "MyTutorialApp", and is deployed in the US-OR 
-region:
+This tutorial is based on the MAUI Flexible Sync Template App named
+``maui.todo.flex``. We start with the default app and build new features
+on it.
 
-.. code-block:: shell
+.. include:: /includes/tutorial-learn-more-template-app.rst
 
-   realm-cli apps create -n MyTutorialApp --template maui.todo.flex \
-      --deployment-model global --location us-or --environment development
+.. tabs::
 
-.. include:: /includes/tutorial-learn-more-template-app.rst
+   .. tab:: App Services UI
```

Comment:
Oops - I think I saw that this update already got merged last week. Maybe rebasing from master will remove this file from this PR?